deploy-testchain.sh

```diff
@@ -4,91 +4,17 @@
 . "${LIB_DIR:-$(cd "${0%/*}/lib"&&pwd)}/common.sh"
 setConfigFile "testchain"
 
-# Send ETH to Omnia Relayer
-OMNIA_RELAYER=$(jq -r ".omniaFromAddr" "$CONFIG_FILE")
-seth send "$OMNIA_RELAYER" --value "$(seth --to-wei 10000 eth)"
+CASE="$LIBEXEC_DIR/cases/$1"
 
-# Deploy Values or Medians + OSMs (if delay > 0) (no solc optimization)
-dappBuild osm
-dappBuild testchain-medians
+test $1 != "" && test ! -f "$CASE" && exit 1
```

Comment:
```suggestion
[[ $# == 0 || ! -f "$CASE" ]] && exit 1
```

Reply:
Sounds good

Reply:
Actually the right condition should be: `[[ $# > 0 && ! -f "$CASE" ]] && exit 1`

Reply:
So only needs to exit if there is a param and it is not a valid case.